include/deal.II/numerics/vector_tools_integrate_difference.templates.h

```diff
@@ -451,8 +451,7 @@ namespace VectorTools
 
       const unsigned int n_components = dof.get_fe(0).n_components();
 
-      Assert(exact_solution.n_components == n_components,
-             ExcDimensionMismatch(exact_solution.n_components, n_components));
+      AssertDimension(exact_solution.n_components, n_components);
```

Comment:
Is this in #12639?

Reply:
No, this PR only changes ``vector_tools_integrate_difference.templates.h``. In #12639 I only touched the file ``vector_tools_interpolate.templates.h``.